plugin/ochttp/example_test.go

```diff
@@ -26,3 +27,17 @@ func ExampleTransport() {
 	}
 	_ = client // use client to perform requests
 }
+
+var usersHandler http.Handler
+
+func ExampleHandler() {
+	// Enables OpenCensus for the default serve mux.
+	http.Handle("/users", usersHandler)
+	log.Fatal(http.ListenAndServe("localhost:8080", &ochttp.Handler{}))
```

Comment:
This one looks alright without a Propagation, but let's modify the comment 
```go
// Enables OpenCensus for the default serve mux.
```
to say
```go
// Enables OpenCensus for the default serve mux and B3 Propagation, the defaults
```

Reply:
Done